core/src/main/java/com/graphhopper/util/Parameters.java

```diff
@@ -33,6 +33,10 @@
          */
         public static final String DIJKSTRA_BI = "dijkstrabi";
         /**
+         * Bidirectional Dijkstra making use of stall-on-demand optimization
+         */
+        public static final String DIJKSTRA_BI_SOD = "dijkstrabisod";
```

Comment:
A new internal parameter is ok for tests and comparison etc, but I wouldn't make this official in the Parameters class as I think we have to make SOD the default :)